src/tests/integration_test_utils.rs

```diff
@@ -126,16 +127,29 @@ pub fn lines_match(expected: &str, have: &str, skip: Option<usize>) {
     assert_eq!(it.next(), None, "more input than expected");
 }
 
-pub struct DeltaTest {
+pub struct DeltaTest<'a> {
     config: config::Config,
+    config_ref: Option<&'a config::Config>,
```

Comment:
I'm not so keen on this technique. I'm thinking that the simplest solution would be to change the signature of `with_config` so that it takes ownership of a `Config` struct:

```rust
    pub fn with_config(config: config::Config) -> Self {
```

and then, in any calling code that needs to re-use the same config twice, perhaps use an anonymous function to create the same config multiple times, e.g.

```rust
        let config =
            || make_config_from_args(&default_wrap_cfg_plus(&["--side-by-side", "--width", "55"]));
        {
            DeltaTest::with_config(config())
            ...
```

There are probably other solutions.

Reply:
I like that idea quite a bit.